activate

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# This script sends a one-off activation message (on first boot-up) to
+# Endless, in order to discover how many Endless OS systems are started
+# across the world.
+# 
+# Note that this does not send any identifiable user data: it only transmits the
+# originally installed operating system version, the current version, the
+# machine vendor & product name. When the first activation message is sent, it
+# contains the machine serial number, but this is not sent again so cannot be
+# used for tracking purposes.
+# 
+# (C) 2014 Canonical Ltd.
+# (C) 2016 Endless Mobile Inc.
+# License: GPL v2 or later
+
+# https://home.endlessm.com/v1/activate
+# 
+# Arguments in the query string will be:
+# 
+#  image (the image name, from the xattr on /sysroot)
+#  release (from /etc/os-release)
+#  vendor (from DMI / device tree)
+#  product (from DMI / device tree)
+#  serial (from DMI / Endless mfgdata)
+#  live (true if this is a live USB, false if not)
+
+set -e
+
+DIR=/var/lib/eos-phone-home
+
+if [ ! -d $DIR -o ! -w $DIR ]; then
```

Comment:
Use chained conditions vs `-o` since it's easier to read/maintain:
`if [ ! -d $DIR ] || [ ! -w $DIR ]`

Also, I'd advise for using `"$VARNAME"` in test evaluations.